### PR TITLE
feat(ant): xspec.force.focus=#none

### LINKED ISSUE #1811: Strict mode to disallow focus
Sometimes you forget to remove a `focus` attribute in your XSpec script. For a release process this can be very bad if the XSpec tests say everything is fine though it isn't.

The main [Maven Plugin](https://github.com/xspec/xspec-maven-plugin-1) fails if a `focus` is set, but since [my plugin](https://github.com/nkutsche/xspec-maven-plugin) is mainly just an Ant wrapper, it will only fail if the Ant process underneath throws an exception. Of course I could create a workaround for this by looking into the XSpec files, but maybe it is also interessting for you to introduce something like a strict mode that causes a failure by `focus` attributes. It might also be of interest to some non-Maven users?

### PR DESCRIPTION
**Pull request by AirQuick**
Closes #1811

This PR adds another test for Ant property `xspec.force.focus`.
`#none` (case sensitive) should always invalidate `//x:scenario/@focus`.

### Follow-Up Tasks

- [ ] Update [Wiki](https://github.com/xspec/xspec/wiki/Running-with-Ant#ant-properties).
      (Given that the intricate feature of `xspec.force.focus` is only necessary for [Oxygen XSpec Helper View](https://github.com/xspec/oXygen-XML-editor-xspec-support), Wiki should mention only `#none`.)

### CODE REVIEW

File: test/end-to-end/cases/force-focus_none.xspec [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Remove focus from all the scenarios -->
+<?xspec-test force-focus=#none?>
+
+<x:description query="x-urn:test:do-nothing" query-at="../../do-nothing.xqm"
+	schematron="../../do-nothing.sch" stylesheet="../../do-nothing.xsl"
+	xmlns:t="http://www.jenitennison.com/xslt/xspec"
```
Comment (galtm) [inline test/end-to-end/cases/force-focus_none.xspec:8]: Is it necessary to bind the `t` prefix when this file doesn't use it directly? The test still passed when I removed this line or changed it to bind the prefix to some other URI.

**Review by galtm — APPROVED** [review]
Looks good. In addition to running the e2e tests locally, I added `-Dxspec.force.focus=#none` to the "Additional Ant arguments" field in the Oxygen transformation scenario dialog, and it had the expected effect of removing focus from all the scenarios.

File: test/end-to-end/cases/force-focus_none.xspec (hunk 1 above)
Comment (AirQuick) [reply test/end-to-end/cases/force-focus_none.xspec:8]: I had the same impression and removed `xmlns:t` initially. However, removing it led to a crash in the XQuery test. It's worth noting that the XSLT compiler offers [a finer namespace treatment](https://github.com/xspec/xspec/blob/f24e6971267e64bcabb035f78eb71f5a1ba5869b/src/compiler/xslt/compile/compile-expect.xsl#L87-L89), which is lacking in the XQuery compiler.

Comment (galtm) [reply test/end-to-end/cases/force-focus_none.xspec:8]: Sorry for the distraction. Now I'm seeing the static error in the compiled query when this declaration isn't in the test. I'm not sure why it seemed to pass when I made my earlier comment; my mistake.

**Pull request merged**